run/shared/setupConfigFiles.sh

```diff
@@ -136,6 +136,11 @@ function set_common_settings() {
 
 	# Remove the first comment character on diagnostics
         sed_ie "s|#'|'|"                               HISTORY.rc
+
+	# Make sure that benchmark simulations read the restart file via
+	# GEOS-Chem and not HEMCO.  This will ensure mass conservation
+	# when the run is broken up into several stages.
```

Comment:
Expand the commend to say "since restart file data is stored as REAL8 but HEMCO reads file data as REAL4"